internal/sdkprovider/service/project/organization_project.go

```diff
@@ -0,0 +1,256 @@
+	"github.com/aiven/go-client-codegen/handler/organizationprojects"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/customdiff"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/samber/lo"
+
+	"github.com/aiven/terraform-provider-aiven/internal/common"
+	"github.com/aiven/terraform-provider-aiven/internal/plugin/util"
+	"github.com/aiven/terraform-provider-aiven/internal/schemautil"
+	"github.com/aiven/terraform-provider-aiven/internal/schemautil/userconfig"
+)
+
+var errProjectStructureChangeNotSupported = fmt.Errorf("modifying organization_id, parent_id, or billing_group_id is not supported - please use Aiven Console at https://console.aiven.io/ to perform this operation")
+
+var aivenOrganizationProjectSchema = map[string]*schema.Schema{
+	"organization_id": {
+		Type:        schema.TypeString,
+		Description: "ID of an organization. Changing this property forces recreation of the resource.",
+		Required:    true,
+		ForceNew:    true,
+	},
+	"project_id": {
+		Type:        schema.TypeString,
+		Required:    true,
+		Description: "New Project ID.",
+	},
+	"billing_group_id": {
+		Type:        schema.TypeString,
+		Required:    true,
+		Description: "Billing group ID to assign to the project. It's required when moving projects between organizations.",
+	},
+	"parent_id": {
+		Type:     schema.TypeString,
+		Optional: true,
+		Computed: true,
+		Description: userconfig.Desc(
+			"Link a project to an [organization, organizational unit,](https://aiven.io/docs/platform/concepts/orgs-units-projects) or account by using its ID.",
+		).Referenced().Build(),
+	},
+	"technical_emails": {
+		Type:        schema.TypeSet,
+		Elem:        &schema.Schema{Type: schema.TypeString},
+		Optional:    true,
+		Description: "The email addresses for [project contacts](https://aiven.io/docs/platform/howto/technical-emails), who will receive important alerts and updates about this project and its services. You can also set email contacts at the service level. It's good practice to keep these up-to-date to be aware of any potential issues with your project.",
+	},
+	"tag": {
+		Description: "Tags are key-value pairs that allow you to categorize projects.",
+		Type:        schema.TypeSet,
+		Optional:    true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"key": {
+					Description: "Project tag key.",
+					Type:        schema.TypeString,
+					Required:    true,
+				},
+				"value": {
+					Description: "Project tag value.",
+					Type:        schema.TypeString,
+					Required:    true,
+				},
+			},
+		},
+	},
+	"account_id": {
+		Computed:    true,
+		Description: "Account ID to where the project belongs.",
+		Type:        schema.TypeString,
+	},
+}
+
+func ResourceOrganizationProject() *schema.Resource {
+	return &schema.Resource{
+		Description:   "Creates and manages an [Aiven project](https://aiven.io/docs/platform/concepts/orgs-units-projects#projects).", //todo: update link
```

Comment:
nit: is this still todo? The link seems to work.

Reply:
removed